xhyve/xhyve.go

```diff
@@ -610,6 +613,49 @@ func (d *Driver) publicSSHKeyPath() string {
 	return d.GetSSHKeyPath() + ".pub"
 }
 
+func readLine(path string) []string {
+	lines := []string{}
+	inFile, err := os.Open(path)
+	if err != nil {
+		log.Debugf("Not able to open %s", path)
+	}
+	defer inFile.Close()
+	scanner := bufio.NewScanner(inFile)
+	scanner.Split(bufio.ScanLines)
+	for scanner.Scan() {
+		lines = append(lines, scanner.Text())
+	}
+	return lines
+}
+
+func extractKernelOptions(d *Driver) error {
```

Comment:
Ah, no, no.
Go language can call `d.` methods if the own function has a `d.` pointer receiver.
Yes, call `d.ResolveStorePath(isoMountPath)` need `d`, but if change to below, also can call all of `d.` methods.

```go
func (d *Driver) extractKernelOptions() error {
```